tests/test_model_update.py

```diff
@@ -0,0 +1,26 @@
+from examples.seismic import demo_model
```

Comment:
We shouldn't test `examples` inside `tests` because in theory the examples folder shouldn't (couldn't) even be accessible.

So I've been thinking about this...

How about we add it somewhere here? https://github.com/devitocodes/devito/blob/master/examples/seismic/tti/tti_example.py#L72

It should then be captured by CI through this line? 
https://github.com/devitocodes/devito/blob/master/.github/workflows/examples.yml#L49 
(@mloubout )

Reply:
I moved the model update test to tti_exemple.py as suggested